LICENSE

```diff
@@ -13,8 +13,8 @@ The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
 
 You may not redistribute the software or modifications as part of any
-application that can be described as a development toolkit, library or
-framework.
+application that can be described as a grid visualization engine development
+toolkit, library or framework.
```

Comment:
Comments on this? (Issue #766) If I understand the original intent correctly, people can use Hypergrid in their project, so long as it's not just a wrapper with a new name slapped on it passing it off as their own.

Adding the phrase "grid visualization engine" is my attempt to make it clear that their project can be a framework, just not solely and specifically a grid framework, with no value added.

Can you think of perhaps a better way of expressing this? Alternatives I considered were:
* "_specifically_ as a grid visualization engine [toolkit...]"
* "_solely and specifically_ as a grid visualization engine [toolkit...]"

Reply:
Hey @joneit We want to use the original MIT License verbatim and not add custom language.

Reply:
OK.... just to confirm, this is a *big* change, right? Just so we are on the same page, the custom paragraph in the LICENSE file **came from Openfin and has been in there forever:**

> You may not redistribute the software or modifications as part of any
application that can be described as a grid visualization engine development
toolkit, library or framework.

Removing the above paragraph entirely will allow anyone to fork it, rename it, and call it their own. I personally don't see that as a good idea, but it's your call!

Please confirm that you want to do this, which would revert it to a plain vanilla MIT license, and I will make the edit in a new commit.

Thx.

Reply:
@joneit Just got confirmation. We want to go MIT all the way